src/tab.rs

```diff
@@ -1689,46 +1738,71 @@ impl Tab {
 
-                let row = if condensed {
-                    widget::row::with_children(vec![
-                        widget::icon::icon(item.icon_handle_list_condensed.clone())
-                            .content_fit(ContentFit::Contain)
-                            .size(icon_size)
-                            .into(),
-                        widget::column::with_children(vec![
-                            widget::text(item.name.clone()).into(),
-                            //TODO: translate?
-                            widget::text(format!("{} - {}", modified_text, size_text)).into(),
-                        ])
-                        .into(),
-                    ])
-                    .align_items(Alignment::Center)
-                    .spacing(space_xxs)
-                } else {
-                    widget::row::with_children(vec![
-                        widget::icon::icon(item.icon_handle_list.clone())
-                            .content_fit(ContentFit::Contain)
-                            .size(icon_size)
-                            .into(),
-                        widget::text(item.name.clone()).width(Length::Fill).into(),
-                        widget::text(modified_text)
-                            .width(Length::Fixed(modified_width))
-                            .into(),
-                        widget::text(size_text)
-                            .width(Length::Fixed(size_width))
-                            .into(),
-                    ])
-                    .align_items(Alignment::Center)
-                    .spacing(space_xxs)
+                let mut columns = vec![widget::icon::icon(item.icon_handle_list.clone())
+                    .content_fit(ContentFit::Contain)
+                    .size(icon_sizes.list())
+                    .into()];
+
+                let column_text = |col: HeadingOptions| match col {
+                    HeadingOptions::Name => item.name.clone(),
+
+                    HeadingOptions::Size => size_text(),
+                    HeadingOptions::Modified => date_format!(modified),
+                    HeadingOptions::Accessed => date_format!(accessed),
+                    HeadingOptions::Created => date_format!(created),
                 };
 
-                let button = widget::button(row)
-                    .width(Length::Fill)
-                    .height(Length::Fixed(row_height as f32))
-                    .id(item.button_id.clone())
-                    .padding(space_xxs)
-                    .style(button_style(item.selected, true))
-                    .on_press(Message::Click(Some(i)));
+                let columns_content = visible_columns.iter().filter(|col| col.active).map(|col| {
+                    widget::text(column_text(col.heading))
+                        .width(get_column_width(col.heading))
+                        .into()
+                });
+                columns.extend(columns_content);
+
+                // let row = if condensed {
```

Comment:
The condensed view is important for being able to adapt to small window sizes. Why is it removed here?